src/Integration/Binding/BindingProcessImpl.cs

```diff
@@ -197,12 +197,6 @@ private BindingConfiguration QueueWriteBindingInformation()
             return configurationPersister.Persist(bound, bindingMode);
         }
 
-        public void PrepareToInstallPackages()
-        {
-            // TODO - CM cleanup
-            // no-op
-        }
```

Comment:
This step and `InstallPackages` are now effectively no-ops, as I'd removed the actual implementations in a previous commit. 

This commit just cleans up `IBindingProcess` interface and top-level implementation by removing the now-redundant steps.